### PR TITLE
MS_Calendar1

### PR DESCRIPTION
**Pull request by tscselma**
Hopefully last version of displaying calendar

### CODE REVIEW

**Comment by tscselma** [comment]
Last version of the Calendar1 function

File: src/test/java/com/meetsky/pages/BasePage.java
Comment (lalesi85) [inline src/test/java/com/meetsky/pages/BasePage.java:1]: Could not see CalendarPage file, instead there 2 features file I think

**Review by lalesi85 — APPROVED** [review]
Just check CalendarPage ..İ could not see. There 2 features files I think

**Review by eyup1312 — APPROVED** [review]

File: src/test/java/com/meetsky/pages/BasePage.java
Comment (tscselma) [reply src/test/java/com/meetsky/pages/BasePage.java:1]: in fact it is available  just at the beginning under the title of MS_CalendarPages

**Review by AlexandruSendrea — APPROVED** [review]

**Review by fahriozcan — APPROVED** [review]

**Pull request merged**